docs/1.guide/2.routing.md

```diff
@@ -50,6 +50,11 @@ Some providers like Vercel use a top-level `api/` directory as a feature, theref
 You will have to use `routes/api/`.
 ::
 
+#### Differences between the two directories
+The `api/` directory is mainly for convenience, routes inside it will be prefixed with `/api/`.
+
+A notable difference between the two directories is how errors are handled by default: in `routes/` they are sent back with with a `Content-Type` of `text/html`, while in `api/` with `application/json`. This behaviour is overridden by some request properties (eg.: `Accept` or `User-Agent` headers).
```

Comment:
It is not difference between directories. any route that starts with `/api` uses JSON content type in errors (both `api/foo.ts` and `server/api/foo.ts` are same behavior)